spec/unit/graph/simple_graph_spec.rb

```diff
@@ -605,7 +609,7 @@ def graph_to_yaml(graph, which_format)
             edge.keys.each { |x| expect(['source', 'target', 'callback', 'event']).to include(x) }
             %w{source target callback event}.collect { |x| edge[x] }
           end
-          expect(Set.new(actual_edge_tuples)).to eq(Set.new(expected_edge_tuples))
+          expect(Set.new(actual_edge_tuples)).to eq(Set.new(expected_edge_tuples.map { |tuple| tuple.map {|e| e.nil? ? nil : e.to_s }}))
```

Comment:
The values are nil, both in the expected tuple and in the hash. The check for nil is there to prevent that a nil becomes an empty string when converting symbols to strings. The test is silly in a way, in that it compares the result of a to_data_hash with the real object. I felt it was easier to just add this conversion than to rewrite the whole test.